common-test-lib/src/main/java/com/google/cloud/tools/intellij/testing/BasePluginTestCase.java

```diff
@@ -75,17 +74,11 @@ public final void setup() {
   protected <N, T extends N> ExtensionPointImpl<T> registerExtensionPoint(
       @NotNull ExtensionPointName<N> name,
       @NotNull Class<T> type) {
-    ExtensionPointImpl<T> extensionPoint = new ExtensionPointImpl<T>(
```

Comment:
`ExtensionPointImpl`'s constructor has been made package private, so we can't use it anymore. Moving forward it's a good idea to avoid depending on impl classes.